site/src/patterns/timeline.md

```diff
@@ -0,0 +1,132 @@
+---
```

Comment:
Could you please add this timeline documentation page to our automated accessibility testing list in `integration/tests/accessibility.spec.ts`, and to our integration testing list in `integration/config/page.ts`?

Reply:
added